core/modules/replica/WorkerSqlRequest.h

```diff
@@ -100,59 +91,83 @@ class WorkerSqlRequest : public WorkerRequest {
 
     /**
      * Extract request status into the Protobuf response object.
-     *
-     * @param response
-     *   Protobuf response to be initialized
+     * @param response The Protobuf response to be initialized
      */
     void setInfo(ProtocolResponseSql& response) const;
 
-    /// @see WorkerRequest::execute
     bool execute() override;
 
 private:
-
-    /// @see WorkerSqlRequest::create()
     WorkerSqlRequest(ServiceProvider::Ptr const& serviceProvider,
                      std::string const& worker,
                      std::string const& id,
                      ProtocolRequestSql const& request);
 
-    /// @return a connector as per the input request
+    /// @return A connector as per the input request
     std::shared_ptr<database::mysql::Connection> _connector() const;
 
     /**
-     * The query generator uses parameters of a request to compose
-     * a desired query.
-     *
-     * @param conn
-     *   a reference to the database connector is needed to process arguments
-     *   to meet requirements of the database query processing engine.
-     *
-     * @return
-     *   a query as per the input request
-     *
-     * @throws std::invalid_argument
-     *   if the input parameters are not supported
+     * The query generator for simple requests uses parameters of a request
+     * to compose a desired query.
+     * @note this method is only used to generate a single query for
+     *   the non-batch requests.
+     * @param conn A reference to the database connector is needed to process
+     *   arguments to meet requirements of the database query processing engine.
+     * @return A query as per the input request.
+     * @throws std::invalid_argument For unsupported requests types supported.
      */
     std::string _query(std::shared_ptr<database::mysql::Connection> const& conn) const;
 
     /**
-     * Initialize the Protobuf response object.
-     * 
-     * @note
-     *   This method is called to extract result set of a query via the connector
-     *   provided and cache the result within the request.
-     * 
-     * @param conn
-     *   a valid database connector for extracting a result set
+     * An alternative query generator for the "batch" requests which involve
+     * a table uses parameters of a request and the name of a table
+     * to compose a desired query. Only a subset of requests can be
+     * executed in the "batch" mode.
+     * @param conn A reference to the database connector is needed to process
+     *   arguments to meet requirements of the database query processing engine.
+     * @param table The name of table affected by the query.
+     * @return A query as per the input request and the name of a table.
+     * @throws std::invalid_argument For unsupported requests types supported.
```

Comment:
`unsupported requests types supported`? Some hard decisions have to made here to unsupport supported 😉